packages/main/src/Text.ts

```diff
@@ -0,0 +1,64 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import renderer, { html } from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import { getScopedVarName } from "@ui5/webcomponents-base/dist/CustomElementsScope.js";
+
+// Styles
+import styles from "./generated/themes/Text.css.js";
+
+/**
+ * @class
+ *
+ * <h3>Overview</h3>
+ *
+ * The Text component displays text that can be used in any content area of an application.
+ *
+ * <h3>Usage</h3>
+ *
+ * - Use the `Text` if uou want to display text inside a form, table, or any other content area.
+ * - Do not use the `Text` if you need to reference input type of components (use ui5-label).
```

Comment:
I suggest:
- Do not use the `ui5-text` if you need to reference the input of other components (use `ui5-label` instead).